lib/membrane_rtc_engine/engine.ex

```diff
@@ -818,6 +900,7 @@ defmodule Membrane.RTC.Engine do
 
       {:present, actions, state} ->
         MediaEvent.create_peer_left_event(peer_id)
+        |> then(&%Message.MediaEvent{rtc_engine: self(), to: :broadcast, data: &1})
```

Comment:
We repeat this creating structs in multiple places. What do you think about looking at something like this?
```elixir
def Message.MediaEvent.new(data, to \\ :broadcast, rtc_engine \\ self())
````

Reply:
I was thinking about this but I am not convinced. If MediaEvent.new contained some logic then I would say yes. But at the moment we are just assigning fields so I decided to keep it as simple as possible